Documentation/ReleaseNotes/NuttX-12.3.0

```diff
@@ -566,7 +566,7 @@ Drivers With Improvements
 * [#10778](https://github.com/apache/nuttx/pull/10778) mmcsd: add get emmc cid register interface.
 * [#10168](https://github.com/apache/nuttx/pull/10168) mmcsd: fix byte_block_count error in byte mode
 * [#10440](https://github.com/apache/nuttx/pull/10440) mmcsd: fix regression causing emmcsd not working
-* [#9937](https://github.com/apache/nuttx/pull/9937) mmcsd: mmcsd_sdio: config timout to write one data block
+* [#9937](https://github.com/apache/nuttx/pull/9937) mmcsd: mmcsd_sdio: config timeout to write one data block
```

Comment:
should we keep the release note as before?

Reply:
I have a temporary project that will turn on Nuttx's spell check, but before that I need to fix all the typographical errors scanned in. Documentation is also covered because there is a pull url mark, so I think it's no problem to fix the typo at the same time.